### ISSUE TITLE
gevent-1.3.0 and bottle 0.12.13

### ISSUE THREAD

**Issue by CrimsonGlory** [bottlepy/bottle#1065]
The reciently relased version of gevent (version 1.3.0.) is crashing bottle (it updated from 1.2.2 to 1.3.0)
```
 INFO:root:Namespace(host='40d6ffd5f91f', port=4546)
 Bottle v0.12.13 server starting up (using GeventServer())...
 Listening on http://40d6ffd5f91f:4546/
 Hit Ctrl-C to quit.

 Traceback (most recent call last):
   File "/myapp/cuckoo_api_mockup.py", line 69, in <module>
     run(host=args.host, port=args.port, server='gevent')
   File "/usr/local/lib/python2.7/site-packages/bottle.py", line 3127, in run
     server.run(app)
   File "/usr/local/lib/python2.7/site-packages/bottle.py", line 2907, in run
     from gevent import wsgi, pywsgi, local
 ImportError: cannot import name wsgi
```

**Comment by agalera** [comment]
if it can be of help, have you tried using gunicorn + gevent? so you can also take advantage of multiple cpu's

maybe I can use this temporarily

**Comment by wumpus** [comment]
Gevent release notes say:

`The long-deprecated and undocumented module gevent.wsgi was removed.`

**Comment by jessekps** [comment]
python 3.x exact same issue, temporary fix  `pip install gevent==1.2.2` also works

**Comment by wumpus** [comment]
It's worth noting that Python 3.7 requires the new gevent>=1.3 that breaks bottle.

If I read the calendar correctly, Python 3.7 candidate 1 goes out in 4 days, and the final 3.7.0 release is scheduled for 19 days from now.

**Comment by and800** [comment]
There's fix in master already: cafc15419cbb4a6cb748e6ecdccf92893bb25ce5

Maybe it should be backported in 0.12 branch?

**Comment by wumpus** [comment]
Looks like it would be trivial to backport it! And looking at the gevent docs, this module gevent.pywsgi has ben in gevent for a long time, so it's safe for earlier versions -- bottle had been using its now removed alias.

**Comment by avelino** [comment]
It would be interesting to contain in the documentation of the bottle, you can send PR (have my approval, just not restoring by @defnull 😄 )

**Comment by braedon** [comment]
What's the chance of getting this backported, and a release cut, any time soon?

**Comment by defnull** [comment]
I'd be happy to accept a PR (against the release-0.12 branch) that adds support for both (old and new) versions of gevent.

**Comment by braedon** [comment]
@defnull does @dhagrow's [PR](https://github.com/bottlepy/bottle/pull/1082) do what you need?

**Comment by defnull** [comment]
Fix was merged. Thanks :)

**Comment by defnull** [comment]
No it was not? Hmm, I'm confused by githubs UI sometimes.

**Comment by defnull** [comment]
I just silently released 0.12.14 with this fix included. I need to update the workflow in my local script and get warm to this again, but the next release should be less work for me and a lot less of a hassle.
